fdb-extensions/src/main/java/com/apple/foundationdb/async/RankedSet.java

```diff
@@ -246,6 +382,80 @@ public RankedSet(Subspace subspace, Executor executor) {
         });
     }
 
+    /**
+     * Removes a key from the set.
+     * @param tc the transaction to use to access the database
+     * @param key the key to remove
+     * @return a future that completes to {@code true} if the set was modified, that is, if the key was present before this operation
+     */
+    public CompletableFuture<Boolean> remove(TransactionContext tc, byte[] key) {
```

Comment:
There is a separate commit to relocate this method, which might make it easier to track how it has changed (other than that).